Create bin directory when building

samfield1 opened the pull request:
Make sure that the bin directory exists. This prevents the mv command from renaming the executables to `bin`.

samfield1 committed:
Create bin directory when building

vosslab commented:
Thanks for the pull request and for addressing the BIN_DIR issue. I know it has been over two years, but I just noticed this. I refactored the Makefile and implemented my own BIN_DIR solution. Appreciate the effort, and feel free to share any other ideas!

samfield1 commented:
Hi,
Thanks for merging this! :) I found your program very convenient for
determining molecular volume.
Best,
Sam Mansfield

On Sat, Nov 23, 2024 at 10:20 AM Neil Voss ***@***.***> wrote:

> Thanks for the pull request and for addressing the BIN_DIR issue. I know
> it has been over two years, but I just noticed this. I refactored the
> Makefile and implemented my own BIN_DIR solution. Appreciate the effort,
> and feel free to share any other ideas!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/vosslab/vossvolvox/pull/3#issuecomment-2495511963>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABCGLZKIHY4BYNLAAYTQD3L2CCMLXAVCNFSM6AAAAABSLFQDUWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIOJVGUYTCOJWGM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
